libraries/load-themed-styles/src/index.ts

```diff
@@ -251,10 +251,33 @@ function applyThemableStyles(stylesArray: ThemableArray, styleRecord?: IStyleRec
 export function loadTheme(theme: ITheme | undefined): void {
   _themeState.theme = theme;
 
+  try {
+    const { style } = document.body;
+    for (const key in theme) {
+      if (theme.hasOwnProperty(key)) {
+        style.setProperty(`--${key}`, theme[key]);
+      }
+    }
+  } catch (e) {
+    // Log but suppress error
+    console.error(e);
```

Comment:
Why suppress?

Reply:
Yeah, thinking just letting the exception bubble might be best.

Reply:
If this API isn't available or something I don't want that to break the page.